src/pages/data-visualization/getting-started/angular.mdx

```diff
@@ -127,12 +115,10 @@ cd packages/angular
 yarn run storybook
 ```
 
-Open browser to `http://localhost:9006/`.
-
-### List of available components
+Launch your browser and go to `http://localhost:9006/`.
```

Comment:
This is the same content that is in the live version of these docs, is there a change we should make?

Reply:
No we should keep it. The reason I mentioned it is because it seems to have been moved to a place where it doesn't belong.
(this is related to my comments above re. the development header)